ubuntudesign/cloud-vanilla-theme
# Update file structure

## Done

- Adds new file structure following from the designers document
- Adds github issue and PR templates

## QA

- Pull down the branch and run ```npm install``` and ```gulp build``` check everything builds and lints correctly and the demo looks correct

## Details

_n/a_

## Screenshots

_n/a_
